src/pages/signin/signin.js

```diff
@@ -1,29 +1,64 @@
-// 요소노드 및 테스트데이터 import
-import { inputEmailEl, inputPswEl, eyeOffIconByPswIcon, showMessageByEmailEl, showMessageByPswEl, TEST_EMAIL, TEST_PSW  } from '../../shared/constants.js'
+import { 
+  inputEmailEl, 
+  inputPswEl, 
+  eyeOffIconByPswIcon, 
+  showMessageByEmailEl, 
+  showMessageByPswEl, 
+  API } 
+  from '../../shared/constants.js'
 
-// 이메일 체크, 이메일 안내메시지 삭제
-import { checkEmailValid, removeEmailCheckMessage } from '../../utils/checkEmail.js';
+import { 
+  checkEmailValid, 
+  removeEmailCheckMessage } 
+  from '../../utils/checkEmail.js';
 
-// 비밀번호 체크, 비밀번호 안내메시지 삭제, 비밀번호 보이기on/off
-import { checkPasswordFill, removePswCheckMessage, showPsw } from '../../utils/checkPsw.js';
+import { 
+  checkPasswordFill, 
+  removePswCheckMessage, 
+  showPsw } 
+  from '../../utils/checkPsw.js';
+
+// 페이지 접근 시 엑세스토큰 보유 -> folder페이지로 이동
+if(localStorage.getItem("accessToken")) {
+  location.href = '../folder/folder.html';
+}
 
 // 로그인 가능 체크
-function checkUserForLogin(e) {
-  const inputEmailValue = inputEmailEl.value;
-  const inputPswValue = inputPswEl.value;
-  if (
-      inputEmailValue === TEST_EMAIL && 
-      inputPswValue === TEST_PSW
-    ) 
-      return location.href = '/folder';
+async function checkUserForLogin() {
+  const inputUserEmail = inputEmailEl.value;
+  const inputUserPsw = inputPswEl.value;
+
+  try {
+    const response = await fetch(`${API}/sign-in`, {
+    method: "POST",
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      email: inputUserEmail,
+      password: inputUserPsw
+    })
+  });
   
-  showMessageByEmailEl.textContent = '이메일을 확인해주세요.';
-  showMessageByPswEl.textContent = '비밀번호를 확인해주세요.';
+  const responseData = await response.json();
```

Comment:
동작을 체크해보면 localStorage에 accessToken 값이 undefined로 만들어져요.
api response를 체크하면
```
{
    "data": {
        "accessToken": "...",
        "refreshToken": "..."
    }
}
```

이런식으로 data가 오고있어요. 즉 data.accessToken으로 접근하셔야 제대로 토큰값을 읽어올수있답니다 :)
비구조화할당으로 간단하게 가져와볼까요?

```suggestion
    const { data } = await response.json();

    if (data) {
      localStorage.setItem("accessToken", data.accessToken);
      location.href = "../folder/folder.html";
    }
```